mantle/cmd/plume/cosa2stream.go

```diff
@@ -38,29 +46,85 @@ var (
 
 	streamBaseURL string
 	streamName    string
+	distro        string
+	output        string
 )
 
 func init() {
 	cmdCosaBuildToStream.Flags().StringVar(&streamBaseURL, "url", "", "Base URL for build")
 	cmdCosaBuildToStream.Flags().StringVar(&streamName, "name", "", "Stream name")
-	cmdCosaBuildToStream.MarkFlagRequired("name")
+	cmdCosaBuildToStream.Flags().StringVar(&distro, "distro", "", "Distribution (fcos, rhcos)")
+	cmdCosaBuildToStream.Flags().StringVar(&output, "output", "", "Use this file as baseline and target for output (default: no baseline, print to stdout)")
```

Comment:
Bikeshed: wonder if e.g. `target` is a better name for this? Since it's actually both an input and an output.

Reply:
OK, renamed.